packages/wrangler/src/config/environment.ts

```diff
@@ -77,8 +77,19 @@ interface EnvironmentInheritable {
 	main: string | undefined;
 
 	/**
-	 * The directory in which module rules should be evaluated in a `--no-bundle` worker
-	 * This defaults to dirname(main) when left undefined
+	 * If true then Wrangler will traverse the file tree below `base_dir`;
+	 * Any files that match `rules` will be included in the deployed worker.
+	 * Defaults to true if `no_bundle` is true, otherwise false.
+	 *
+	 * @inheritable
+	 */
+	find_additional_modules: boolean | undefined;
```

Comment:
What happens if something is both statically imported by the entrypoint, and matched with a module rule?

Reply:
This can be seen in the `additional_modules` fixture where the `text.txt` file is statically imported.